### PR TITLE
Integration-specs-PullRequest

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
**About this project**<br/>

This part of the project is about creating integration tests in our Blog App.

**For it:** <br/>
-A branch named integration-specs was created.
- We write integration tests with gem named capybara.
- We secure app from N + 1 problem.
- We updated files and created a folder named views inside spec folder.
- We created spec files and these files were stored inside view folder.
- Linters were passed.
- We respected the gitflow.

**Here are our screenshots of before solving some issues :**<br/>

Before the Fix: <br/>
integrationTest-before

After the Fix: <br/>
IntegrationTests-after
integrationTestSecondPicture

Thanks for the review,

Best regards,

### CODE REVIEW

File: spec/views/user_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    it 'displays the number of posts for each user' do
+      visit user_path(@doctor)
+      expect(@doctor.posts.count).to have_content(4)
+      visit user_path(@advent_hospital_patient)
+        .expect(@advent_hospital_patient.posts.count).to have_content(1)
+    end
+
+    it 'displays the user\'s bio' do
+      visit user_path(@doctor)
+      expect(@doctor.bio).to have_content('Mastology Doctor')
```
Comment (rloterh) [inline spec/views/user_show_spec.rb]: - [ ] A number of the view specs are failing for user show and post index views, let's ensure all test are passing!

**Failing tests:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Ivonne, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 
🏁 🚦  ⛪ 🚜  🏃‍♀️  🌴  🏫 🌳 🏘️ 

## Highlights 👍  
- N + 1 problem addressed!
- Added required integration tests!
- ReadMe looks great!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
Let's make the ReadMe stand out even better by including appropriate instructions under the _Getting Started_ section, on how to set up project, run app and test cases, for instance, using the command `rails server`, `rspec` and so on.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/views/user_show_spec.rb (hunk 1 above)
Comment (IvonneBenitesRodriguez) [reply spec/views/user_show_spec.rb]: Dear @rloterh 👋🏾,
Very nice to meet you,
thank you for your suggestions,
I appreciate it, I already modified it.
Best wishes,

**Review by Efiamotu-1 — APPROVED** [review]
Hi @IvonneBenitesRodriguez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

image

## Highlights :dart: 
* Correct Gitflow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**